en/deploy/signup_pythonanywhere.md

```diff
@@ -1,13 +1,13 @@
 PythonAnywhere is a service for running Python code on servers "in the cloud". We'll use
 it for hosting our site, live and on the Internet.
 
-We will be hosting the blog we're building on PythonAnywhere. Sign up for a "Beginner" account on PythonAnywhere (the free tier is fine, you don't need a credit card).
+We will be hosting the blog we're building on PythonAnywhere. Sign up for a "Beginner" account on PythonAnywhere. (The free tier is fine, you don't need a credit card.)
 
 * [www.pythonanywhere.com](https://www.pythonanywhere.com/)
 
 ![The PythonAnywhere signup page showing button to create a the free 'Beginner' account](../deploy/images/pythonanywhere_beginner_account_button.png)
 
-> **Note** When choosing your username here, bear in mind that your blog's URL will take the form `yourusername.pythonanywhere.com`, so choose either your own nickname or a name for what your blog is all about. Also, be sure to remember your password (add it to your password manager, if you use one).
+> **Note** When choosing your username here, bear in mind that your blog's URL will take the form `yourusername.pythonanywhere.com`, so choose either your own nickname or a name for what your blog is all about. Also, be sure to remember your password. (Add it to your password manager, if you use one.)
```

Comment:
And here

Reply:
Suggest as above:

Also, be sure to remember your password -- add it to your password manager, if you use one.